### PR TITLE
ADEN-10434 UCP new startup POC

### PR DESCRIPTION
**Pull request by Bielik20**

### CODE REVIEW

File: platforms/ucp/ucp-platform.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+			UcpA9ConfigSetup,
+			UcpDynamicSlotsSetup,
+			UcpSlotsStateSetup,
+			CommonBiddersStateSetup,
+			UcpTemplatesSetup,
+			UcpBillTheLizardSetup,
+			LabradorSetup,
+			TrackingSetup,
+			AdEngineRunnerSetup,
+			() => communicationService.dispatch(adEngineConfigured()),
```
Comment (fraszczakszymon) [inline platforms/ucp/ucp-platform.ts:57]: It seems the order has changed. Previously the event was dispatched before wiki context setup, am I right?

Comment (Bielik20) [reply platforms/ucp/ucp-platform.ts:57]: I feel like it was after configure: https://github.com/Wikia/ad-engine/blob/f144e3d9bd51c0c722ae55eeeaca182496fc4aa0/platforms/ucp/index.ts#L20-L21

Comment (fraszczakszymon) [reply platforms/ucp/ucp-platform.ts:57]: Sorry, I mixed configure with run method. You're right

File: platforms/ucp/ucp-platform.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+export class UcpPlatform {
+	constructor(private pipeline: ProcessPipeline, private noAdsDetector: NoAdsDetector) {}
+
+	execute(): void {
+		// Config
+		this.pipeline.add(
+			() => context.extend(basicContext),
+			parallel(UcpIocSetup, () => bootstrapAndGetConsent()),
+			UcpWikiContextSetup,
+			() => context.set('state.isMobile', false),
```
Comment (Loczi94) [inline platforms/ucp/ucp-platform.ts:43]: Does it have to be here? 🙈  Maybe we can move it to UcpBaseContextSetup?

**Pull request merged**